src/main/java/uk/nhs/hee/tis/trainee/forms/repository/AbstractCloudRepository.java

```diff
@@ -91,17 +93,19 @@ public T save(T form) {
       PutObjectRequest request = PutObjectRequest.builder()
           .bucket(bucketName)
           .key(key)
-          .metadata(Map.of(
-              "id", form.getId().toString(),
-              "name", fileName,
-              "type", "json",
-              "formtype", form.getFormType(),
-              "lifecyclestate", form.getLifecycleState().name(),
-              SUBMISSION_DATE,
-              form.getSubmissionDate().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME),
-              "traineeid", form.getTraineeTisId(),
-              "deletetype", DeleteType.PARTIAL.name(),
-              "fixedfields", FIXED_FIELDS
+          .metadata(Map.ofEntries(
+              entry("id", form.getId().toString()),
+              entry("name", fileName),
+              entry("type", "json"),
+              entry("isarcp", form.getIsArcp().toString()),
+              entry("programmemembershipid", form.getProgrammeMembershipId().toString()),
```

Comment:
Why do we need this in the metadata?
The metadata should be generic for all forms, but these fields are only needed for Form Rs

Reply:
for it to be read in the formr view in admins, when speaking with Steve he said it would need to be in the custom metadata for the FE to be able to read it.